test/functions.js

```diff
@@ -694,6 +694,9 @@
       assert.deepEqual(_.toArray(cb(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)), _.range(1, 11));
     });
 
+    var deepProperty = _.iteratee(['a', 'b']);
+    assert.equal(deepProperty({a: {b: 2}}), 2, 'treats an array as a deep property accessor');
+
```

Comment:
The Tweet responses add some interesting color as well. Thanks for running that @jdalton. That being said, I don't think this thread is actually about (re)considering the string syntax. It seems pretty clear to me that the string syntax is not something Underscore wants to support.